app/MMB/src/components/Alert.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text, StyleSheet, Icon } from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+
+type CommonProps = {|
+  title: React.Element<typeof Translation>,
+  children: React.Node,
+|};
+
+type Props = {|
+  type: 'danger' | 'warning' | 'success',
+  ...CommonProps,
+|};
+
+const AlertDanger = (props: CommonProps) => (
+  <View style={[styles.container, styles.containerDanger]}>
+    <View style={styles.iconContainer}>
+      <Icon name="warning" size={18} color={RED} />
+    </View>
+    <View>
+      <Text style={[styles.titleText, styles.titleDanger]}>{props.title}</Text>
+      <View style={styles.body}>{props.children}</View>
+    </View>
+  </View>
+);
+
+export default function Alert(props: Props) {
```

Comment:
Is this component necessary? Cannot we just use the "danger alert" directly?

Reply:
The idea was to expose one component, with 3 types, then that component selects the necessary one. 

Basically these components will be the same, with different styling.